frontend/ui/widget/dictquicklookup.lua

```diff
@@ -106,7 +143,7 @@ function DictQuickLookup:update()
         w = Screen:getWidth(),
         h = Screen:getHeight(),
     }
-    if self.word_box then
+    if self.word_box and not not self.fullpage then
```

Comment:
Obviously, i had been too negative there :) I'll remove one of these `not` at next changes request (dunno if one started reviewing this, and if one can diff commits, if i commit a fix for that now)